Yoast/wordpress-seo
# Add tests for HowTo Schema generator

## Context


* We want to have the test coverage of the indexables code to 100%.

## Summary


This PR can be summarized in the following changelog entry:

* Improves the test coverage of the HowTo Schema generator to 96%.

## Relevant technical choices:

* An `elseif` statement in `generators/schema/howto.php` is seen as being covered, although its contents *are* seen as covered.  

## Test instructions

This PR can be tested by following these steps:

* Check that Travis passes.
* Run the tests locally, including coverage.
* Check that the coverage of `generators/schema/howto.php` is 96%.

## UI changes

## Documentation

## Quality assurance


Fixes #
